src/models/info.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseModel } from "./base";
+import { Contact } from "./contact";
+import { License } from "./license";
+
+export class Info extends BaseModel {
+    title(): string {
+        return this.json("title");
+    }
+
+    version(): string {
+        return this.json("version");
+    }
+
+    description(): string {
+        return this.json("description");
+    }
+
+    termsOfService(): string {
+        return this.json("termsOfService");
+    }
+
+    contact(): Contact {
+        const doc = this.json("contact");
+        return doc && new Contact(doc);
+    }
+
+    license(): License {
```

Comment:
We have change return type to `Contact | undefined`. Why? Because some fields are optional, so in TS tools people will have info that they should check if value exist or not, if you return only `Contact` you tell: `Hey, you will have always Contact instance` and it's not true.

I don't know how we will handle situation for v2 and v3, example: `channels` will be optional from v3 (in 99%, yes?) so in v2 people will have always list channels, but in v3 they will have undefined or list. We have to discuss it.

```suggestion
    contact(): Contact | undefined {
        const doc = this.json("contact");
        return doc && new Contact(doc);
    }

    license(): License | undefined {
```